bspump/ipc/stream_server_source.py

```diff
@@ -18,7 +18,13 @@
 class StreamServerSource(Source):
 
 	ConfigDefaults = {
+
+		# Specify the address with port to listen on
+		'listen': '',
+
+		# For backward compatibility (listen is the latest config option)
 		'address': '127.0.0.1 8888',  # IPv4, IPv6 or unix socket path
+
```

Comment:
The default must be with `listen`, not with address.
Address is not to be listed here at all.